tools/emacs/README.txt

```diff
@@ -21,3 +22,14 @@ This file may also be called ~/.emacs.el, or ~/.emacs.d/init.el
 
 (1) Copy lammps.el to the directory ~/.emacs.d
 
+=Update:
+========
+
+The package may now also be installed by a QUELPA style recipe, namely:
+
+```lisp
+(use-package lammps-mode
+  :recipe (:fetcher url :url "https://github.com/lammps/lammps/blob/master/tools/emacs/lammps.el"))
```

Comment:
Plese note, that it is generally not a good idea to refer to the master branch for software that people should be using for production. Best is likely to refer to the stable branch (we'll have a new stable release in a few weeks, so this update would be included there), or at least the unstable branch, which corresponds to the patch releases posted by Steve.

Reply:
Sure, actually I will update this with the MELPA repository recipe, so it's best not to merge yet...